share/spice/graphical_calculator/graphical_calculator.js

```diff
@@ -0,0 +1,144 @@
+(function (env) {
+
+  "use strict";
+
+  /*
+    $.getScript("http://localhost:5000/share/spice/graphical_calculator/js/d3/3.5.6/d3.js")
```

Comment:
You can use `DDG.get_asset_path()` (https://duck.co/duckduckhack/spice_js_api#getassetpathspicename-asset) to load these libs.

That way you won't need to mess with DuckPAN or anything else :)